### ISSUE TITLE
the problem of test set act as Validation set

### ISSUE THREAD

**Issue by crush-kunkun** [Frank-Wang-oss/TS-GAC#2]
` train_loss, train_acc, train_loss_details = model_train(model, temporal_contr_model, model_optimizer, temp_cont_optimizer, criterion, train_dl, config, device, training_mode, lambda1, lambda2, lambda3,
                                                                num_remain_aug1, num_remain_aug2)
        valid_loss, valid_acc, _, _ = model_evaluate(model, temporal_contr_model, test_dl, device, training_mode)
        if training_mode != 'self_supervised':  # use scheduler in all other modes.
            scheduler.step(valid_loss)`
In your code, I noticed that it seems that you will directly use the test set to calculate validation losses and participate in learning rate tuning, which does not quite comply with regulations. And after I changed your code to a validation set, the accuracy decreased by two points. Can you explain this issue
